src/convertToObject.js

```diff
@@ -1,12 +1,30 @@
-'use strict';
-
 /**
- * @param {string} sourceString
+ * @param {string} stylesString
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
+function convertToObject(stylesString) {
+  const result = {};
+
+  // Разделение на ключ-значение
+  stylesString.split(';').forEach((line) => {
```

Comment:
Consider adding a check to ensure `stylesString` is a string before proceeding with the split operation. This will make the function more robust against invalid inputs.